tests/functional/algos/test_algos.py

```diff
@@ -12,6 +12,7 @@
 
 config_files = ['random_config.yaml']
 fidelity_config_files = ['random_config.yaml', 'asha_config.yaml']
+hyperband_config = ['hyperband.yaml']
```

Comment:
Can you add the yaml file to the list above instead so that we don't need to add a test explicitly?